src/types/api.js

```diff
@@ -113,5 +117,12 @@
  * @prop {boolean} canLeave
  */
 
+/**
+ * @typedef Analytics
+ * @prop {() => any} sendPageView
+ * @prop {(action: string, label?: string, value?: number) => void} track
+ * @prop {Object} events
```

Comment:
This service interface definition isn't part of the Hypothesis API. Can you move it to `src/sidebar/services/analytics.js`?